### PR TITLE
Adding CIN to the abbreviations list

### PR DESCRIPTION
**Pull request by ChrisHesketh**
# <Describe your Changes>
Adding CIN to the abbreviations list

## Why? What? How?

* Why are you making this change?
* Briefly explain your changes.
* Optionally explain how you are making your changes, you can skip this if you aren't doing anything special.

## Checklist

- [ ] This PR **does not** alter any page content.
- [ ] This PR introduces a new page.
  - [ ] I have ensured that there are links from other pages to this new content.
- [X ] This PR updates existing pages.
- [ ] I have previewed my changes using [StackEdit](https://stackedit.io/app), and they match my expectations.
- [X] I have checked that any abbreviations used match up with [built-in abbreviations](https://github.com/CPS-Innovation/digital-sop/tree/main/snippets/abbreviations.md).

### CODE REVIEW

File: snippets/abbreviations.md [hunk 1]
```diff
@@ -12,6 +12,7 @@
 *[BAU]: Business as Usual
 *[SOP]: Standard Operating Procedures
 *[CMS]: Case Management System
+*[CIN]: **C**MS **IN**tegration
```
Comment (acodeninja) [inline snippets/abbreviations.md:15]: Unfortunately the abbreviations do not render markdown, so you'll want to replace this with `Case Management System Integration` to make it work.

Also another unfortunate thing is that the abbreviations only work with a full word, so `CIN1` will not use this abbreviation, you'll want to add `*[CIN1]: Case Management System Integration One`

Comment (ChrisHesketh) [reply snippets/abbreviations.md]: Done

**Review by acodeninja — APPROVED** [review]

File: snippets/abbreviations.md [hunk 2]
```diff
@@ -12,8 +12,13 @@
 *[BAU]: Business as Usual
 *[SOP]: Standard Operating Procedures
 *[CMS]: Case Management System
+*[CIN1]: Case management system Integration environment One
```
Comment (chrisns) [inline snippets/abbreviations.md:15]: inconsistent capitalisation

Comment (chrisns) [inline snippets/abbreviations.md:15]: There **isn't** a CIN1
There **is** a CIN5
there is also a PT for performance testing
PCMS = production CMS
LCMS = ??
MO = Model Office CMS

**Review by chrisns — CHANGES_REQUESTED** [review]

**Pull request merged**